test/python/v1/nexus/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
         - MY_POD_IP=10.0.0.2
         - NEXUS_NVMF_ANA_ENABLE=1
         - NEXUS_NVMF_RESV_ENABLE=1
-    command: ${SRCDIR}/target/debug/io-engine -g 0.0.0.0 -l 1 -r /tmp/ms0.sock
+    command: ${SRCDIR}/target/debug/io-engine -g 0.0.0.0 -l 1,2 -r /tmp/ms0.sock
```

Comment:
you might want to use 5 or increment the other ones, otherwise this core 2 will be shared with ms1